From e9126b30c2ef1e96ca5df6d8844747821ccd146f Mon Sep 17 00:00:00 2001
From: Liu Ying <victor.liu@nxp.com>
Date: Thu, 17 Aug 2017 13:55:53 +0800
Subject: [PATCH 2390/5242] MLK-16207-2 drm/bridge: it6263: 2nd time to
 workaround cable detection failure

commit  6e9cab3daf043e8c62e03bdcd70490d8fde29a07 from
https://source.codeaurora.org/external/imx/linux-imx.git

There is cable detection failure issue on i.MX8qxp MEK platform at boot
time when we use single LVDS to HDMI display.  The workaround is to read
the cable detection status for even more times.  Based on experiments, it
looks reading for 90 times works.

Signed-off-by: Liu Ying <victor.liu@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/gpu/drm/bridge/it6263.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/gpu/drm/bridge/it6263.c b/drivers/gpu/drm/bridge/it6263.c
index 5094c66..7b39421 100644
--- a/drivers/gpu/drm/bridge/it6263.c
+++ b/drivers/gpu/drm/bridge/it6263.c
@@ -377,7 +377,7 @@ static void it6263_reset(struct it6263 *it6263)
 	 * cable detection failure issue at boot time on some
 	 * platforms.
 	 */
-	for (i = 0; i < 40; i++)
+	for (i = 0; i < 90; i++)
 		regmap_read(it6263->hdmi_regmap, HDMI_REG_SYS_STATUS, &status);
 
 	return (status & HPDETECT) ? connector_status_connected :
-- 
1.7.9.5

